### PR TITLE
fix: recent import and export records remove interval

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## Does this PR introduce a user interface change?
- [ ] Yes(screenshot is required)
- [x] No

## Which versions should be patched?
release/1.1

## Which issue(s) this PR fixes:

Fixes #  recent import and export records remove interval

### CODE REVIEW

File: shell/app/modules/project/pages/test-manage/case/header/import-export-record.tsx [hunk 1]
```diff
@@ -75,13 +75,6 @@ const ImportExportRecord = () => {
     getData(true);
   });
 
-  useInterval(
-    () => {
-      getData(false);
-    },
-    hasError ? null : 5000,
-  );
```
Comment (daskyrk) [inline shell/app/modules/project/pages/test-manage/case/header/import-export-record.tsx:83]: not remove directly.

File: shell/app/modules/project/pages/test-manage/case/header/export-file.tsx [hunk 2]
```diff
@@ -27,6 +27,7 @@ const ExportFile = () => {
         i18n.t('application:The export task has been created, please check the progress in the record'),
         4,
       );
+      afterExport?.();
```
Comment (McDaddy) [inline shell/app/modules/project/pages/test-manage/case/header/export-file.tsx:30]: if u add `?` means afterExport maybe undefined, but in the props definition, afterExport is required

File: shell/app/modules/project/pages/test-manage/case/header/import-export-record.tsx [hunk 3]
```diff
@@ -36,6 +36,7 @@ const ImportExportRecord = ({
   const [contentVisible, setContentVisible] = useState(false);
   const userMap = useUserMap();
   const [hasError, setHasError] = useState(false);
+  const [isfinished, setIsfinished] = useState(false);
```
Comment (McDaddy) [inline shell/app/modules/project/pages/test-manage/case/header/import-export-record.tsx:39]: no spell check alert?

File: shell/app/modules/project/pages/test-manage/case/header/import-export-record.tsx [hunk 4]
```diff
@@ -44,6 +45,10 @@ const ImportExportRecord = ({
   const getData = (firstTime: boolean) => {
     getImportExportRecords(['import', 'export'])
       .then((result: TEST_CASE.ImportExportRecordItem[]) => {
+        if (!result.find((item) => !['success', 'fail'].includes(item.state))) {
```
Comment (McDaddy) [inline shell/app/modules/project/pages/test-manage/case/header/import-export-record.tsx:48]: too many negative check.
` if (!result.find((item) => ['pending', 'processing'].includes(item.state)))`
or
` if (result.every((item) => ['success', 'fail'].includes(item.state)))`

File: shell/app/modules/project/pages/test-manage/case/manual-test.tsx [hunk 5]
```diff
@@ -46,6 +46,7 @@ const ManualTest = () => {
   const [enhanceFilterVisible, setEnhanceFilterVisible] = React.useState(false);
   const [showRefresh, setShowRefresh] = React.useState(false);
   const [justImportSetId, setJustImportSetId] = React.useState<number | null>(null); // to cache the test set id which current user just Import
+  const [importExportRecordKey, setImportExportRecordKey] = React.useState<number>(0);
```
Comment (McDaddy) [inline shell/app/modules/project/pages/test-manage/case/manual-test.tsx:49]: if u pass `0` as init value ,  ts will infer this state as number, no need to add generic

File: shell/app/modules/project/pages/test-manage/case/manual-test.tsx [hunk 6]
```diff
@@ -169,6 +174,7 @@ const ManualTest = () => {
             />
             <BatchProcessing recycled={query.recycled === 'true'} />
             <ImportExportRecord
+              key={importExportRecordKey}
```
[the remark this answers is not in the document]
Comment (hujiahao-hjh) [reply shell/app/modules/project/pages/test-manage/case/manual-test.tsx:177]: for rerender the component when the key is changed.

**Pull request merged**